src/components/UserDetailSections/PatronBlock/PatronBlock.js

```diff
@@ -158,9 +160,23 @@ class PatronBlock extends React.Component {
       },
     } = this.props;
 
+    const pointerWrapper = (content, isManual) => (
+      isManual
+        ? <div className={css.pointerWrapper}>{content}</div>
+        : content
+    );
+
     return {
-      'Type': f => f?.type ?? <FormattedMessage id="ui-users.blocks.columns.automated.type" />,
-      'Display description': f => f.desc || f.message,
+      'Type': f => {
+        const type = f?.type ?? <FormattedMessage id="ui-users.blocks.columns.automated.type" />;
+
+        return pointerWrapper(type, f?.type);
+      },
+      'Display description': f => {
```

Comment:
@Dmitriy-Litvinenko apologies for asking you about it as it looks like the original code had that already too but I'm wondering if we could use some sort of key here instead of english versions for column names like `Display description` etc. 

You can review this code:
https://github.com/folio-org/ui-erm-comparisons/blob/aee68fa3eda3c81b1ba4058d54d86424a7ae0de0/src/components/EntitlementsAgreementsList/EntitlementAgreementsList.js#L21-L38

I think it can be achieved with the `columnMapping`

Reply:
On my opinion we should do it out of scope current pull request. Technical task was created https://issues.folio.org/browse/UIU-2091 .
Could you please review it?
CC: @zburke @annamelnyk